src/js/components/OptionalPortsAndServiceDiscoveryComponent.jsx

```diff
@@ -159,9 +159,8 @@ var OptionalPortsAndServiceDiscoveryComponent = React.createClass({
 
     var fieldTooltipMessage = (
       <span>
-        Port configuration for applications in Marathon can be confusing,
-        to Learn more visit the
-        docs: <a href={ExternalLinks.PORTS} target="_blank">Read more</a>.
+        Enter the port you want to assign to your
+        VIP. <a href={ExternalLinks.PORTS} target="_blank">Read more</a>.
```

Comment:
VIPs should only be mentioned if the feature is available. Also, I frankly think this message is a bit misleading because it's not clear we're talking about the Container Port or the VIP Port. @sascala can we generalise this message so it simply points to a page where we can explain things a bit better?